app/src/main/java/com/dessalines/thumbkey/ui/components/settings/behavior/BehaviorActivity.kt

```diff
@@ -149,47 +208,46 @@ fun BehaviorActivity(
                             minSwipeLengthState,
                             slideSensitivityState,
                             slideEnabledState,
+                            slideCursorMovementModeState,
+                            slideSpacebarDeadzoneEnabledState,
+                            slideBackspaceDeadzoneEnabledState,
                             autoCapitalizeState,
                             spacebarMultiTapsState,
                         )
                     },
                 )
-                val minSwipeLengthStr = stringResource(
-                    R.string.min_swipe_length,
-                    minSwipeLengthState.value.toInt().toString(),
-                )
-                SettingsSlider(
-                    valueRange = 0f..200f,
-                    state = minSwipeLengthState,
+                SettingsList(
+                    state = slideCursorMovementModeState,
+                    enabled = slideEnabledState.value,
```

Comment:
Great job on disabling this list!

Reply:
Woooo! finally closed it haha.
Yeah I was thinking of opening an issue following this, since there's lots of opinions and changes to make, which we can tackle one by one.
And well done to github for not telling me it's been merged, even though I have my emails set to do so lol.